src/panels/MissionsPanel/MissionContent.tsx

```diff
@@ -0,0 +1,228 @@
+        .filter((v) => v !== undefined);
+    }
+
+    // We want to add any actions related to the whole mission
+    const phasesToCheck: (Phase | Milestone)[] = [missionOverview];
+
+    // Add any extra actions related to the currently shown phase or milestone
+    if (
+      displayedPhase.data &&
+      displayedPhase.data.actions &&
+      displayedPhase.data !== missionOverview
+    ) {
+      phasesToCheck.push(displayedPhase.data);
+    }
+
+    const result = phasesToCheck.flatMap((phase) =>
+      findCurrentActions(phase, allActions)
+    );
+
+    setCurrentActions(result);
+  }, [allActions, displayedPhase, missionOverview]);
+
+  function getTitle() {
+    // Hide title if the overview is currently shown
+    const isShowingOverview = displayedPhase.data?.name === missionOverview.name;
+    const hasType = displayedPhase.type;
+    const hideTile = isShowingOverview || !hasType;
+    return hideTile ? '' : `${displayedPhase.type}: ${displayedPhase.data.name}`;
+  }
+
+  function getTimeString() {
+    switch (displayedPhase.type) {
+      case DisplayType.Phase: {
+        const start = new Date(displayedPhase.data.timerange.start).toDateString();
+        const end = new Date(displayedPhase.data.timerange.end).toDateString();
+        return `${start} - ${end}`;
+      }
+      case DisplayType.Milestone:
+        return new Date(displayedPhase.data.date).toDateString();
+      default:
+        return '';
+    }
+  }
+
+  function setPhaseManually(phase: DisplayedPhase) {
+    setDisplayedPhase(phase);
+    setDisplayCurrentPhase(false);
+  }
+
+  return (
+    <Container my={'xs'}>
+      <Group grow wrap={'nowrap'} align={'start'}>
+        <TimeLine
+          allPhasesNested={allPhasesNested.current}
+          displayedPhase={displayedPhase}
+          missionOverview={missionOverview}
+          setDisplayedPhase={setPhaseManually}
+        />
+        <div style={{ maxWidth: 'none' }}>
+          <Group grow gap={'xs'}>
+            <Button
+              onClick={() =>
+                setPhaseManually({ type: DisplayType.Phase, data: missionOverview })
+              }
+            >
+              Overview
+            </Button>
+            <Button
+              onClick={() => setDisplayCurrentPhase((prevState) => !prevState)}
+              variant={displayCurrentPhase ? 'filled' : 'light'}
+            >
+              Current Phase
+            </Button>
+          </Group>
+          {displayedPhase.data ? (
+            <>
+              <Text>{title}</Text>
+              <Text c={'dimmed'}>{timeString}</Text>
+              <Text my={'xs'}>{displayedPhase.data.description}</Text>
+              {displayedPhase.data.link && (
+                <Button onClick={() => console.log(displayedPhase.data.link)}>
+                  TODO: fix Read more
+                </Button>
+              )}
+              {displayedPhase.data.image && (
```

Comment:
THere seems to be something wrong with this check, as there are cases when it is rendered even if there is no image. 

E.g. in Apollo profile: 
![image](https://github.com/user-attachments/assets/e1492ced-60b2-461b-b5ca-79296dbfcfd8)

Reply:
In this case the link is dead and gives 404: https://www.nasa.gov/sites/default/files/styles/full_width_feature/public/thumbnails/image/s69-31739orig.jpg